API.md

```diff
@@ -792,6 +783,36 @@ Or, if you never expect Nativefier to open an "external" page in your OS browser
 nativefier https://google.com --internal-urls ".*?"
 ```
 
+##### Internal Login Pages
+*[Breaking change in Nativefier 43.0.0]* Finally, URLs for known login pages
+are considered internal. This does not replace `internal-urls`, it complements
+it, and happens *before* your `internal-urls` rule is applied. So, if you
+already set the flag to let such auth pages open internally, you don't need to
+change it but it might be unnecessary.
+
+We think this is desirable behavior and are so far unaware of cases where users
+might not want this. If you disagree, please chime in at
+[PR #1124: App: Automatically consider known login pages as internal](https://github.com/nativefier/nativefier/pull/1124)
+
```

Comment:
We didn't get any negative feedback about this since it landed, time to scrap the call for feedback 🙂

```suggestion
```